server/import.go

```diff
@@ -46,14 +47,24 @@ func ImportServer(configuration, settings, event, eventRules, entrylist, bop, as
 	return nil
 }
 
-func importFile(reader io.Reader, config interface{}, filename string) error {
+func importFile(reader io.ReadSeeker, config interface{}, filename string) error {
 	r := transform.NewReader(reader, utf16Encoding.NewDecoder().Transformer)
 
 	if err := json.NewDecoder(r).Decode(config); err != nil {
-		logrus.WithError(err).
-			WithField("file", filename).
-			Error("Error unmarshalling configuration file JSON on import")
-		return err
+		if _, er := reader.Seek(0, io.SeekStart); er != nil {
+			return er
+		}
+
+		// trying decode non ut16 content
+		if err2 := json.NewDecoder(reader).Decode(config); err2 != nil {
```

Comment:
You can just redeclare "err" here. The `if err := ...` construct has its own scope.

Reply:
If I do it, I will loose the main error